api/config/default.php

```diff
@@ -343,27 +343,28 @@
 	'homepageOrderjdownloader' => '18',
 	'homepageOrderunifi' => '19',
 	'homepageOrderPihole' => '20',
-	'homepageOrdertautulli' => '21',
```

Comment:
please keep the default ordering and place the new at 42

Reply:
fixed in newest commit